voa_composition:
  ros__parameters:
    timeout_s_pc: 3.000
    pass_x_min: 0.0
    pass_x_max: 3.0
    pass_y_min: -1.5
    pass_y_max: 1.5
    leaf_size: 0.05

    timeout_s_gm: 3.000
    height: 6
    width: 4
    resolution: 0.05
    use_cv: True
    use_cuda: True
    is_dilate: True
    dilate_kernel_size: 3
    is_erode: False
    erode_kernel_size: 3

    timeout_s_sc: 3.000
    kMaxOdomBufferLength: 10
    kMaxPredictTimes: 5
    kPredictDtS: 1.0
    kSafetyDtS: 0.3
    kRobotLengthM: 0.65
    kRobotWidthM: 0.45
    kObstatleHeight: 0.10
    kP4Vx: 0.010
    kP4Vyaw: 0.010
